azure-devops/golang/stages/40-delivery/dynamic-go-arm.yaml

```diff
@@ -26,6 +26,10 @@ stages:
               cacheHitVar: 'AZURE_DEPLOY_CACHE_HIT'
             displayName: 'Cache for Resources Deployment in Azure'
 
+          - script: |
+              echo "##vso[task.setvariable variable=AZURE_DEPLOY_CACHE_HIT;isOutput=true]$(AZURE_DEPLOY_CACHE_HIT)"
```

Comment:
Take a look in the other YAML files, when we do this output, the variable name is not uppercase. We have to be consistent. What is going to be the standard? Uppercase or Lowercase for outputs? And what is the standard for environment variables? Uppercase or Lowercase? And so on...